sys-kernel/coreos-kernel/coreos-kernel-4.3.0.ebuild

```diff
@@ -7,3 +7,21 @@ inherit coreos-kernel
 
 DESCRIPTION="CoreOS Linux kernel"
 KEYWORDS="amd64 arm64"
+
+DEPEND="
```

Comment:
Also, if any of them have sub-slots you can add := to trigger the rebuild, similar to the bootengine dependency in the eclass.